tests/pyTigerGraphUnitTest.py

```diff
@@ -42,6 +43,7 @@ def make_connection(graphname: str = None):
         certPath=server_config["certPath"],
         sslPort=server_config["sslPort"],
         gcp=server_config["gcp"],
+        jwtToken=server_config["jwtToken"]
```

Comment:
Can we support this in our current testing pipeline?

Reply:
Yeah, we need to modify the config file to add the jwtToken field but set it to empty str to avoid actually using it now. 

Meanwhile, we need to figure out with QE team how to config the DB to use jwtToken

Reply:
Just double check here: does server_config contains this key "jwtToken"? I don't see the config file modified.

Reply:
Sure. I just add the jwtToken to the config file and set it to empty. 

> Just double check here: does server_config contains this key "jwtToken"? I don't see the config file modified.